### PR TITLE
Add shortcuts to the media dialog

### PR DESCRIPTION
**Pull request by charlag**
Hey
I have no idea what I'm doing but seems to be working and I really want this :sweat_smile:

### CODE REVIEW

**Comment by charlag** [comment]
Oops, sorry for the issues. I've fix styling issues and force-pushed but it doesn't seem to be picked up by CI?..

**Comment by nolanlawson** [comment]
I'll rebase and give this another go. I'm in the middle of a big rewrite on the MediaDialog, but I see what you're trying to do and it LGTM. :) Thanks!

File: src/routes/_components/dialog/components/MediaDialog.html [hunk 1]
```diff
@@ -198,6 +209,18 @@
           this.scrollToItem(i, true)
         }
       },
+      next () {
+        let { scrolledItem } = this.get()
+        if (scrolledItem < 4) {
```
Comment (nolanlawson) [inline src/routes/_components/dialog/components/MediaDialog.html]: this should be `length` instead of 4, since there may be less than 4 media.

**Comment by nolanlawson** [comment]
Thanks again! :)

**Pull request merged**